### ISSUE TITLE
Details of training

### ISSUE THREAD

**Issue by AdanMichael** [JJGO/UniverSeg#36]
dear authors , i would like to konw how you set learning rate , batch size and epoch when you train the model . Could you share with me ?
